index.bs

```diff
@@ -1394,21 +1404,38 @@ To <dfn>parse filter data</dfn> given a |value|:
 Issue: Determine whether to limit [=string/length=] or
 [=string/code point length=] for |filter| and |d| above.
 
+To <dfn>parse filter config</dfn> given a |value|:
+
+1. If |value| is not a [=map=], return null.
+1. Let |lookbackWindow| be null.
+1. If |value|["`lookback_window`"] [=map/exists=]:
+    1. If |value|["`lookback_window`"] is not a non-negative integer, return null.
+    1. Set |lookbackWindow| to the [=duration=] of |value|["`lookback_window`"] seconds.
+    1. [=map/Remove=] |value|["`lookback_window`""].
```

Comment:
This is elegant, but we have to decide if we're OK breaking any registrations that are already using the `lookback_window` key.

Related to that, we currently fail source registrations if their filter data contains a `source_type` key. Should we do something similar here?

Reply:
If we do break registrations that are using the `lookback_window` key, it does makes sense to fail source registrations if their filter data contains a `lookback_window` key, it will prevent confusion down the line.

However, whether this is fine to do is a good question. Was there a similar decision when `source_type` was added ?

Reply:
IIRC `source_type` was added at the same time as filtering in general, so we were able to prevent that from the start, at least in the spec (#468).

Reply:
I added a note in the description on the backward incompatibility. We can also mitigate the impact for currently stored sources by having a transition release that supports test lookback_window key.